sycl/test-e2e/ProgramManager/uneven_kernel_split.cpp

```diff
@@ -1,4 +1,4 @@
-// REQUIRES: gpu, opencl-aot, ocloc
+// REQUIRES: cpu, gpu, opencl-aot, ocloc
```

Comment:
I think this will always be `false`. Did you mean `gpu || cpu`?

Reply:
No, this test uses `cpu` and `gpu` at the same time

```
// RUN: %clangxx -fsycl -fsycl-targets=spir64_x86_64,spir64_gen -Xsycl-target-backend=spir64_gen %gpu_aot_target_opts %t.o %t1.o %t2.o -o %t.out

```

Arvind is on vacation, so adding some other people to review instead

Reply:
> No, this test uses `cpu` and `gpu` at the same time

I don't understand how it is supposed to work in our LIT infrastructure. We have this:
https://github.com/intel/llvm/blob/8761d403787f32348377d88e5def1aa37dccae8e/sycl/test-e2e/lit.cfg.py#L774-L777

so `cpu`/`gpu` is a per-device property. It can't be both at the same time. What do I miss?

Reply:
The test requires both the OCL CPU runtime installed and an Intel GPU. Do we have a way to check for that in our infrastructure? If not, we may need to disable the test and ask the owner to rework it.

Reply:
https://github.com/intel/llvm/blob/8761d403787f32348377d88e5def1aa37dccae8e/sycl/test-e2e/lit.cfg.py#L657-L661 maybe...

Reply:
ah ok nice thanks